test/extended/images/mongodb_ephemeral.go

```diff
@@ -24,8 +24,14 @@ var _ = g.Describe("[images][mongodb] openshift mongodb image", func() {
 			g.By("creating a new app")
 			o.Expect(oc.Run("new-app").Args("-f", templatePath).Execute()).Should(o.Succeed())
 
-			g.By("expecting the mongodb service get endpoints")
-			o.Expect(oc.KubeFramework().WaitForAnEndpoint("mongodb")).Should(o.Succeed())
+			g.By("waiting for the deployment to complete")
+			err := exutil.WaitForADeployment(
```

Comment:
Now this could use your new helper `WaitForADeploymentToComplete`? And I suspect most callers of `WaitForADeployment` are actually interested in `WaitForADeploymentToComplete` :yum: